example.con

```diff
@@ -0,0 +1,8 @@
+let A be a point;
+let B be a point;
+let C1 be a circle with center A and edge B;
```

Comment:
There are indeed multiple ways to construct the same circle, but I think this is most true to the classical definition of a circle